h5p-bildetema/src/components/Header/Header.stories.tsx

```diff
@@ -47,9 +37,9 @@ const Template = (isWordView: boolean): JSX.Element => {
 };
 
 export const Default = (): JSX.Element => {
-  return Template(false);
+  return Template();
 };
 
 export const isWordView = (): JSX.Element => {
-  return Template(true);
+  return Template();
 };
```

Comment:
Can we remove this, as it seems identical to the `Default`?

Reply:
Yes!